### ISSUE TITLE
Sequences are not generated with jooq version 3.15.2 but are generated with 3.14.13

### ISSUE THREAD

**Issue by nbottarini** [jOOQ/jOOQ#12435]
### Expected behavior
Generate sequences.
### Actual behavior
Class Sequences is not generated

### Steps to reproduce the problem
I'm using jooq with gradle plugin v6.0.1. The problem only occurs after upgrading from jooq 3.14.13 to 3.15.x. The sequences are generated correctly in 3.14.13

- If the problem relates to code generation, please post your code generation configuration

```groovy
jooq {
    version.set("3.15.2")

    configurations {
        create("main") {
            generateSchemaSourceOnCompilation.set(true)
            jooqConfiguration.apply {
                logging = org.jooq.meta.jaxb.Logging.WARN
                jdbc.apply {
                    driver = "org.postgresql.Driver"
                    url = "XXXX"
                    user = "XXXX"
                    password = "XXXX"
                }
                generator.apply {
                    name = "org.jooq.codegen.DefaultGenerator"
                    strategy.name = "org.jooq.codegen.DefaultGeneratorStrategy"
                    database.apply {
                        name = "org.jooq.meta.postgres.PostgresDatabase"
                        inputSchema = "public"
                        includes = ".*"
                        excludes = ""
                        recordTimestampFields = "updated_at"
                    }
                    generate.apply {
                        isDeprecated = false
                        isRecords = true
                        isImmutablePojos = true
                        isFluentSetters = true
                    }
                    target.apply {
                        packageName = "XXXX"
                        directory = "generated"
                    }
                }
            }
        }
    }
}
```

### Versions

- jOOQ: 3.15.2
- Java: 16
- Database (include vendor): PostgreSQL
- OS: Windows
- JDBC Driver (include name if inofficial driver): postgresql official driver

**Comment by lukaseder** [comment]
Thanks for your message.

I'm not aware of such a problem, but it is likely you're referring to "system sequences", i.e. sequences that PostgreSQL generated for you for your identity columns? As it is usually not desirable to generate such system objects, they are turned off by default, but you can turn them back on again, using the `<includeSystemSequences/>` flag, see also:

https://www.jooq.org/doc/latest/manual/code-generation/codegen-advanced/codegen-config-database/codegen-database-include-object-types/

That flag was not yet available in jOOQ 3.14:

https://www.jooq.org/doc/3.14/manual/code-generation/codegen-advanced/codegen-config-database/codegen-database-include-object-types/

I hope this helps

**Comment by nbottarini** [comment]
Thanks!!!
